packages/shared/src/index.ts

```diff
@@ -18,3 +18,17 @@ module.exports = {
     '/wrong-entry',
   ],
 };
+
+export const getRemotes = (isServer: boolean) => {
```

Comment:
Could we please completely remove this package?

`getRemotes` is never used.

`mfRouters` can be placed directly in `apps/` folder OR its content can be copy pasted in every _app.tsx file. I think 2nd option is better.

`mfRemote` is designed for generating in runtime (it can be dynamic and contains different URLs for remotes), and putting it in the package completely broke and hide this idea from newcomers.

Reply:
Fixed 👍